UM/Qt/Bindings/Theme.py

```diff
@@ -32,36 +35,90 @@ def __init__(self, engine, parent = None):
         self._em_height = int(QFontMetrics(QCoreApplication.instance().font()).ascent())
         self._em_width = self._em_height;
 
-        self._initializeDefaults()
+        Preferences.getInstance().addPreference("general/theme", Application.getInstance().getApplicationName())
+
+        try:
+            theme_path = Resources.getPath(Resources.Themes, Preferences.getInstance().getValue("general/theme"))
+            self.load(theme_path)
+        except FileNotFoundError:
+            pass
```

Comment:
I see. I guess that the `QCoreApplication.instance().setFont(default_font)` call would change the font in the QCoreApplication which then lets `_initializeDefaults` initialize the system font differently.